### ISSUE TITLE
Copy instead of forward?

### ISSUE THREAD

**Issue by Swiftnesses** [adderou/telegram-forward-bot#4]
Hello,

Great bot, is it possible to copy messages, instead of forwarding them?

**Comment by adderou** [comment]
Yes, you could extract the text inside the messages (If I remember well, its `message.text`) and send it to the chats instead to forwarding it. In the practice, you should change the lines with bot.forwardMessage(...) (138 and 140) to bot.sendMessage(chat_id, message). The text method could only work with text messages, so if you need to copy a photo, audio or video message, you should do some research (I recommend you to check the [Telepot Documentation](https://telepot.readthedocs.io/en/latest/)).
